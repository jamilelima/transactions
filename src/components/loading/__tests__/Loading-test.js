import React from 'react';
import Loading from '../index';
import {create} from 'react-test-renderer';

describe('Test EmptyList component', () => {
  const emptyList = create(<Loading />);
  test('Match with snapashot', () => {
    expect(emptyList.toJSON()).toMatchSnapshot();
  });
});
