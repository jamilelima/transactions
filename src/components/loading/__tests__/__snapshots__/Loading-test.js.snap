// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test EmptyList component Match with snapashot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#1B232F",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "paddingBottom": 60,
        "paddingLeft": 60,
        "paddingRight": 60,
        "paddingTop": 60,
      },
    ]
  }
>
  <Text
    style={
      Object {
        "transform": Array [
          Object {
            "rotate": "0deg",
          },
        ],
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#FFF",
            "fontSize": 64,
          },
          undefined,
          Object {
            "fontFamily": "FontAwesome",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ÔÑê
    </Text>
  </Text>
</View>
`;
